posthog/schema.py

```diff
@@ -457,7 +461,7 @@ class DayItem(BaseModel):
         extra="forbid",
     )
     label: str
-    value: Union[str, int]
+    value: Union[str, AwareDatetime, int]
```

Comment:
I don't really understand how the changes to `schema.ts` result in this `schema.py`. I only see the addition of DatetimeDay, but it's not used anywhere. Is the `schema.ts` not fully committed or am I missing something?

Reply:
Good catch. Seems like it went missing at some point 🤔 

I extended the type in line schema.ts 1187 to include datetime as well. Will fix.

Reply:
Ah, now I have the underlying reason: I put another string type, since ultimately I want string and date-time. But this gets autofixed because it's twice string. 🙃 
Need to find a workaround.